.vsts-ci/azure-pipelines-release.yml

```diff
@@ -31,155 +34,127 @@ trigger:
       - /LICENSE.txt
       - /CODE_OF_CONDUCT.md
 
-jobs:
-- job: 'ReleaseBuild'
-  displayName: Release Build
-  pool:
-    vmImage: 'vs2017-win2016'
-  steps:
-  - template: templates/ci-general.yml
-
-- job: 'SignBuild'
-  displayName: Signing Build
-  dependsOn: 'ReleaseBuild'
-  pool:
-    name: 'Package ES CodeHub Lab E'
-    demands: DotNetFramework
-  steps:
-  - powershell: |
-      Get-ChildItem -Path env:
-    displayName: Capture environment
-    condition: succeededOrFailed()
-
-  - task: PkgESSetupBuild@10
-    displayName: 'Package ES - Setup Build'
-    inputs:
-      productName: GraphicalTools
-
-  - task: DownloadBuildArtifacts@0
-    displayName: 'Download Build Artifacts'
-    inputs:
-      downloadType: specific
-
-  - task: PowerShell@1
-    displayName: 'Extract build zip'
-    inputs:
-      scriptType: inlineScript
-      inlineScript: |
-        Expand-Archive -Path "$env:BUILD_ARTIFACTSTAGINGDIRECTORY\GraphicalTools\Microsoft.PowerShell.GraphicalTools-Windows_NT.zip" -DestinationPath "$env:BUILD_ARTIFACTSTAGINGDIRECTORY\Microsoft.PowerShell.GraphicalTools"
+resources:
+  repositories:
+  - repository: ComplianceRepo
+    type: github
+    endpoint: ComplianceGHRepo
+    name: PowerShell/compliance
+
+stages:
+- stage: Build
+  displayName: Build
+  jobs:
+  - job: 'ReleaseBuild'
+    displayName: Release Build
+    pool:
+      vmImage: 'vs2017-win2016'
+    steps:
+    - template: templates/ci-general.yml
+  - job: 'SignBuild'
```

Comment:
You need to run component governance here or upload assets.json file from here and download them in compliance stage.

Reply:
done